backend/app/tests/services/llm/test_chain.py

```
@@ -0,0 +1,356 @@
+        block1 = ChainBlock(config=config, index=0, context=context)
+        block2 = ChainBlock(config=config, index=1, context=context)
+        block1.link(block2)
+
+        with patch(
+            "app.services.llm.chain.chain.execute_llm_call"
+        ) as mock_execute, patch.object(context, "on_block_completed"):
+            mock_execute.return_value = BlockResult(
+                response=text_response, usage=text_response.usage
+            )
+
+            result = block1.execute(query)
```

Ruff (0.15.2)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)